### DISCUSSION TITLE
Capacitor Plugin API Doc Generator

### DISCUSSION THREAD

**Discussion by adamdbradley** [ionic-team/capacitor#3634] [General]
Hey Capacitor plugin authors! New tool to generate docs straight from your JSDoc comments. A lot like TypeDoc, but specifically for updating README markdown docs and for cap plugins apis. Official plugins will be using this, but open for all plugins to use. Please give it a shot and let us know how it goes, thanks!

https://www.npmjs.com/package/@capacitor/docgen

**Comment by hemangsk** [comment]
Thank you so much!  was looking for something exactly like this :)
